### PR TITLE
mnemonics: new Esperanto word list

### PR DESCRIPTION
**Pull request by moneromooo-monero**
Word list authored by: dnaleor, Engelberg, ProkhorZ
Sources:
  Baza Radikaro Oficiala
  Reta Vortaro (http://www.reta-vortaro.de/revo/)
  Esperanto Panorama - Esperanto-English Dictionary (http://www.esperanto-panorama.net/vortaro/eoen.htm)
  ESPDIC - Paul Denisowski (http://www.denisowski.org/Esperanto/ESPDIC/espdic.txt)

### CODE REVIEW

File: src/mnemonics/esperanto.h [hunk 1]
```diff
@@ -0,0 +1,1695 @@
...
+      "mjelo",
+      "mnemoniko",
+      "mobilizi",
+      "mocio",
+      "moderna",
+      "mohajro",
+      "mokadi",
+      "molaro",
+      "momento",
+      "monda",
```
Comment (anonimal) [inline src/mnemonics/esperanto.h]: Is there a technical reason why we shouldn't have "monero" in the word list?

File: src/mnemonics/esperanto.h [hunk 2]
```diff
@@ -0,0 +1,1695 @@
...
+      "kojoto",
+      "kokoso",
+      "koloro",
+      "komenci",
+      "kontrakto",
+      "kopio",
+      "korekte",
+      "kosti",
+      "kotono",
+      "kovri",
```
Comment (anonimal) [inline src/mnemonics/esperanto.h]: 😄

File: src/mnemonics/esperanto.h (hunk 1 above)
Comment (moneromooo-monero) [inline src/mnemonics/esperanto.h]: None.

**Comment by moneromooo-monero** [comment]
Monero is now in the list :)

File: src/mnemonics/esperanto.h [hunk 3]
```diff
@@ -0,0 +1,1695 @@
...
+      "zirkonio",
+      "zodiako",
+      "zoeto",
+      "zombio",
+      "zono",
+      "zoologio",
+      "zorgi",
+      "zukino",
+      "zumilo",
+      }), 4)
```
Comment (glv2) [inline src/mnemonics/esperanto.h]: In this word list, isn't the unique prefix length 3 instead of 4 ?

**Pull request merged**